Code/Editor/CMakeLists.txt

```diff
@@ -248,39 +248,16 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
                 Gem::AtomToolsFramework.Static
         RUNTIME_DEPENDENCIES
             Gem::LmbrCentral
+            Gem::Camera.Editor
```

Comment:
question: I'm not sure if this is actually needed?

Reply:
The original code was loading that gem, but I removed it and the test is still passing, so I will remove it

Reply:
Okay cool thanks! :+1: